docs/examples/generatedCanonicalFields.json

```diff
@@ -0,0 +1,822 @@
+[
+  {
+    "label": "Org Name",
+    "shortCode": "Org Name",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Mission Statement",
+    "shortCode": "Org Mission Statement",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Website",
+    "shortCode": "Org Website",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Entity Type",
+    "shortCode": "Org Entity Type",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Registration Number: EIN / TIN",
```

Comment:
Similar style guide comment -- should we use `:` or should we use `()` for this type of thing.

Maybe this should be `Organization Tax ID Number (TIN / EIN)

Reply:
What do you think of naming conventions being worked out in the original spreadsheets rather than in downstream processing?